airflow/opt/providers/etna/etna/dags/decorators.py

```diff
@@ -70,7 +88,7 @@ def instantiate_dag(fn):
             schedule_interval=interval,
             default_args=dict(
                 owner="administration",
-                retries=3,
+                retries=10,
```

Comment:
There is already some back off logic built into airflow, so raising the retries just helps persist jobs that have intermittent failures (uploads, network issues)